Dockerfile

```diff
@@ -61,9 +60,16 @@ RUN wget -O yolov5n.pt https://github.com/ultralytics/yolov5/releases/download/v
 # Download some files that are pulled in, so we can run w/o network access
 RUN mkdir -p /root/.config/Ultralytics/ && wget -O /root/.config/Ultralytics/Arial.ttf https://ultralytics.com/assets/Arial.ttf
 
+
+# copy in updates to yolov5 to fix stdout
```

Comment:
Rather than have these files here, it's probably better to:

* Fork the ultralytics/yolov5 repo
* Create a branch at the commit we use (23701ea)
* Apply your changes there
* Update this Dockerfile to pull from our repo/branch instead

That will make it much easier to see the changes you've made, etc. in case we need to upgrade in the future.

Reply:
Same applies with the requirements changes—best to make them in a fork of the yolov5 repo rather than in here.

Reply:
These COPY lines need removing